src/components/wavyWave/wave.js

```diff
@@ -58,6 +83,27 @@ export function createWaves() {
   heightChanger.innerText = "Change Height";
   heightChanger.onclick = () => changeHeightOf("eq-line");
 
-  container.append(waveBox, colorChanger, heightChanger);
+  const startButton = createElement("button", {
+    className: "start-button",
+    innerText: "Start",
+    onclick: () => {
+      startAnimation();
+    },
+  });
+  const stopButton = createElement("button", {
+    className: "stop-button",
+    innerText: "Stop",
+    onclick: () => {
+      stopAnimation();
+    },
+  });
```

Comment:
instead of using two buttons why not using one
```suggestion
const startButton = createElement("button", {
    className: "start-button",
    innerText: "Start",
    id: "loopBtn",
    onclick: () => {
      playStopButton();
    },
  });
```